Src/BootCamp.Chapter/BootCamp.Chapter.csproj

```diff
@@ -5,4 +5,8 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.10.0" />
```

Comment:
This should not be included in Chapter project, it's only needed for testing.

Reply:
Uninstalled FluentAssertions package.